tests/singlerod/short/singlerod_short.omn

```diff
@@ -16,8 +16,8 @@ input "singlerod_short.rtk.xml"
 !################################# PHYSICS ####################################
 
 [PHYSICS=ce ce]
-ce_lib ce_v71
-ce_pole_lib_path "/data/pole_data/pole.h5"
+ce_lib_path "/opt/scale/h5_data/ce_v7.1_endf.h5"
```

Comment:
Did it not work for you with `ce_lib` set to the shortcut instead of specifying the full path?  If the data path is set correctly when you configure/build SCALE, the shortcut version _should_ work.

Reply:
Ah, yes, I just had the HDF5 data in a different directory than that used at configure time. Once I added a symlink, the shortcut version works now :+1:

Reply:
Also, I didn't realize that the `.inp.xml` file has absolute paths -- I'll get rid of that one